opus/import/obs_bundle_cassini_uvis_solarocc_beckerjarmak2023.py

```
@@ -0,0 +1,244 @@
+        return self._north_based_angle_helper()[0]
+
+    def field_obs_ring_geometry_north_based_incidence2(self):
+        return self.field_obs_ring_geometry_north_based_incidence1()
+
+    def field_obs_ring_geometry_north_based_emission1(self):
+        return self._north_based_angle_helper()[1]
+
+    def field_obs_ring_geometry_north_based_emission2(self):
+        return self.field_obs_ring_geometry_north_based_emission1()    
+
+    # Would be –rings:observed_ring_elevation, but that should become negative after 2009 and doesn't, so there may be errors in the Becker/Jarmak labels for rings:observed_ring_elevation 
```

Ruff (0.14.5)

Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)